LEGO1/mxhashtable.h

```diff
@@ -9,6 +9,11 @@
 #define HASH_TABLE_OPT_EXPAND_ADD         1
 #define HASH_TABLE_OPT_EXPAND_MULTIPLY    2
 
+template <class T>
+class MxHashTableCursor;
+template <class T>
+class MxList;
```

Comment:
Is it supposed to be `MxHashTable`?

Reply:
Only `MxHashTableCursor` is needed here. I got confused by the missing forwarded declarations in `mxlist.h`.